### DISCUSSION TITLE
[SUGGESTION] Add Claimer for Auction

### DISCUSSION THREAD

**Discussion by TGWaffles** [HypixelDev/PublicAPI#459] [Ideas]
It could be really useful to add whoever claimed the auction to the /auctions_ended endpoint, so that in co-ops it's possible to get who claimed the co-op auction. 

Example: Players A, B, and C are in a co-op. Player A creates an auction - it sells, and player B claims it while player A is offline. Player A is now confused what happened to their auction. This variable would give player B's uuid in the auction_ended as claimer, or seller_claimer. A bot/website could keep track of these and allow player A to find out what happened to the auction.

**Comment by Harry-McLean** [comment]
This would be really useful for me, would this be possible?
